draft-ietf-jsonpath-base.md

```diff
@@ -1171,15 +1192,23 @@ The following table lists filter expression operators in order of precedence fro
 |:--:|:--:|:--:|
 |  5  | Grouping | `(...)` |
 |  4  | Logical NOT | `!` |
-|  3  | Relations | `==`&nbsp;`!=`<br>`<`&nbsp;`<=`&nbsp;`>`&nbsp;`>=`<br>`=~`<br>` in ` |
+|  3  | Relations | `==`&nbsp;`!=`<br>`<`&nbsp;`<=`&nbsp;`>`&nbsp;`>=`<br>`=~` |
 |  2  | Logical AND | `&&` |
 |  1  | Logical OR | `¦¦`   |
 {: title="Filter expression operator precedence" }
 
 #### Semantics
 {: unnumbered}
 
-The `filter-selector` works with arrays and objects exclusively. Its result might be a list of *zero*, *one*, *multiple* or *all* of their element or member values then. Applied to other value types, it will select nothing.
+The `filter-selector` works with arrays and objects exclusively. Its result is a list of *zero*, *one*, *multiple* or *all* of their array elements or member values, respectively. Applied to other value types, it will select nothing.
+
+A relative path, beginning with `@`, refers to the current array element or member value as the
+filter selector iterates over the array or object.
+
+Comparisons using one of the operators `<`, `<=`, `>`, and `>=` are between numeric values only.
+Using these operators to compare other types of values produces a "false" comparison result.
+
+The semantics of regular expressions is as defined in {{-iregexp}}.
```

Comment:
"is" should be "are", semantics is plural

Reply:
se·man·tics | səˈmæn(t)ɪks |
plural noun [usually treated as singular]

(But I won't contradict native speakers here...)

Reply:
https://www.collinsdictionary.com/dictionary/english/semantics claims that "semantics" functions as singular in British English, is unclear about American English, but then goes on to cite the example "They are also logically meaningful, they have well-defined truth conditions, and their semantics are unambiguous." This sounds plain wrong to my British English ear, but the spec should be in American English, so I'll change it.

Reply:
Fixed (after rebasing) with the observation that "semantics" is already plural elsewhere in the spec.